### PR TITLE
Add components redux setup and basic routing

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <nav className="nav-bar navbar-container mb-0">
+      <div className="nav-innerContainer font-clr">
+        <h2 className="nav-heading mr-4 nav-custom-sty">Social Cloud.</h2>
+      </div>
+
+      <div className="nav-innerContainer font-clr width-auto">
+        <input className="nav_searchBar" type="text" />
+        <span className="searchBar_icon">
+          <i className="fa-solid fa-magnifying-glass"></i>
```
Comment (Hamza-777) [inline src/components/Navbar/Navbar.jsx:14]: Try using react-icons package, it contains a lot more icons and all free.

**Review by Hamza-777 — APPROVED** [review]
Everything looks good.

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/components/Navbar/Navbar.jsx:14]: Sure I will look into it

**Review by anik31 — APPROVED** [review]
Everything looks good, nice work.

suggestion - you can change the sidebar navigation texts

**Comment by abhisheksahani-nova** [comment]
> Everything looks good, nice work.
> 
> suggestion - you can change the sidebar navigation texts

Yaah almost forgot that

**Pull request merged**